### PR TITLE
Add Alert Dismiss

### PR DESCRIPTION
**Pull request by tonyhhyip**

### CODE REVIEW

File: lib/xhp-bootstrap/alert.php [hunk 1]
```diff
@@ -33,6 +34,14 @@ protected function render(): XHPRoot {
     $ret->addClass('alert');
     $ret->addClass('alert-'.$this->:use);
 
+    if ($this->:dimiss) {
+        $close =
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+          </button>;
+	$this->prependChild($close);
```
Comment (BobertForever) [inline lib/xhp-bootstrap/alert.php]: It looks like you have some spacing issues. The tabbing convention for this project is two spaces; you seem to use both 3 and 4 here.

**Comment by fredemmott** [comment]
Please can you fix the whitespace issue mentioned.

**Comment by tonyhhyip** [comment]
Issue Fixed

File: lib/xhp-bootstrap/alert.php [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
   attribute
     :div,
+    bool dimiss = false,
```
Comment (fredemmott) [inline lib/xhp-bootstrap/alert.php]: typo both here and in the example

Comment (tonyhhyip) [inline lib/xhp-bootstrap/alert.php]: More detail please

Comment (fredemmott) [inline lib/xhp-bootstrap/alert.php]: s/dimiss/dismiss/

**Comment by tonyhhyip** [comment]
Fix Spelling Mistake

**Comment by fredemmott** [comment]
Thanks :)

**Pull request merged**